tutorials/source-modeling/plot_beamformer_lcmv.py

```diff
@@ -0,0 +1,169 @@
+# activity on the trials with left auditory stimulation. Note that
+# beamformers are usually computed in a :class:`volume source space
+# <mne.VolSourceEsitmate>`, as a visualization
+# of only the surface activation can misrepresent the data.
+
+data_path = sample.data_path()
+subjects_dir = data_path + '/subjects'
+raw_fname = data_path + '/MEG/sample/sample_audvis_filt-0-40_raw.fif'
+fwd_fname = data_path + '/MEG/sample/sample_audvis-meg-vol-7-fwd.fif'
+
+# Read forward model
+forward = mne.read_forward_solution(fwd_fname)
+
+# Read the raw data
+raw = mne.io.read_raw_fif(raw_fname, preload=True)
+raw.info['bads'] = ['MEG 2443']  # bad MEG channel
+
+# Set up the epoching
+event_id = 1  # those are the trials with left-ear auditory stimuli
+tmin, tmax = -0.2, 0.5
+events = mne.find_events(raw)
+
+# pick relevant channels
+raw.pick(['meg', 'eog'])  # pick channels of interest
+
+# Create epochs
+proj = False  # already applied
+epochs = mne.Epochs(raw, events, event_id, tmin, tmax,
+                    baseline=(None, 0), preload=True, proj=proj,
+                    reject=dict(grad=4000e-13, mag=4e-12, eog=150e-6))
+
+# Visualize averaged sensor space data
+evoked = epochs.average()
+evoked.plot_joint()
+
+###############################################################################
+# Computation of covariance matrices
+# ----------------------------------
+# The spatial filter is computed from two ingredients: the forward model
+# solution that we read from disk above and the covariance matrix of the data.
+# The data covariance matrix will be `inverted`_ during the spatial filter
+# computation, so it is valuable to plot the covariance matrix and its
+# eigenvalues to gauge whether matrix inversion will be possible.
+# Also, because we want to combine different channel types (magnetometers and
+# gradiometers), we need to account for the different amplitude scales of these
+# channel types. To do this we will supply a noise covariance matrix to the
+# beamformer, which will be used for whitening.
+
+data_cov = mne.compute_covariance(epochs, tmin=0.05, tmax=0.25,
+                                  method='empirical')
+noise_cov = mne.compute_covariance(epochs, tmin=tmin, tmax=0,
+                                   method='empirical')
+
+data_cov.plot(epochs.info)
+
+###############################################################################
+# Compute the spatial filter
+# --------------------------
+# Now we can compute the spatial filter.
+# When looking at the covariance matrix plots, we can see that our data is
+# slightly rank-deficient. Thus, we will regularize the covariance matrix by
+# setting the parameter ``reg=0.05``. This corresponds to loading the
+# diagonal of the covariance matrix with 5% of the sensor power.
+#
+# Different variants of the LCMV beamformer exist. We will compute a
+# unit-noise-gain beamformer, which normalizes the beamformer weights to take
+# care of an inherent depth bias. To achieve this, we set
```

Comment:
see prev. comment: depth bias should be introduced in "background" or "intro" section if possible; that will lead to some changes in wording here.

Reply:
I did not move that up, as it is not a problem of beamformers but the forward model. I tried to make it more clear though.